libs/common/src/lib/interfaces/benchmark.interface.ts

```diff
@@ -6,6 +6,7 @@ export interface Benchmark {
   performances: {
     allTimeHigh: {
       performancePercent: number;
+      date: Date;
```

Comment:
Can you move up to keep the alphabetical order?

Reply:
done